pkg/reconciler/broker/resources/brokercell.go

```diff
@@ -28,7 +28,7 @@ import (
 // TODO(#866) Delete hard-coded brokercell once we can dynamically assign brokercell to brokers.
 const DefaultBrokerCellName = "default"
 
-func CreateBrokerCell(b *v1beta1.Broker) *inteventsv1alpha1.BrokerCell {
+func CreateBrokerCell(_ *v1beta1.Broker) *inteventsv1alpha1.BrokerCell {
```

Comment:
This might be unexpectedly complex, but should we also change this to sth like: `CreateBrokerCell(_ *CellTenant)`, just in case in the future we want multiple brokercell support.?

Reply:
OK, I am guessing this will be in your next PR, right? If not, maybe we should add another TODO here.

Reply:
Added a TODO.

Looking at the subsequent PRs, it turns out I just pass `nil`.

Reply:
I guess when we support multiple BrokerCell, we can then make the change.